dev/run-tests.py

```diff
@@ -495,7 +494,7 @@ def main():
               " install one and retry.")
         sys.exit(2)
 
-    java_version = determine_java_version(java_exe)
+    #java_version = determine_java_version(java_exe)
```

Comment:
Looks a mistake.

Reply:
This line isn't used, but I forgot a space after the comment. I'll just delete this.